roaringbitmap.h

```diff
@@ -23,6 +23,7 @@
 #include "libpq/pqformat.h"
 
 /* must include "roaring.h" before redefine malloc functions */
+#define CROARING_ATOMIC_IMPL 3
```

Comment:
Is this just for the stuff in `portability.h`?

Reply:
removed in https://github.com/BrandwatchLtd/pg_roaringbitmap/pull/3